swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -6870,16 +7011,32 @@ components:
             $ref: "#/components/schemas/paymentInitiationBulkElement_json"
         transactionStatus:
           $ref: "#/components/schemas/transactionStatus"
+        tppMessage:
+          type: array
+          items: 
+            $ref: "#/components/schemas/tppMessageGeneric"
+          description: Messages to the TPP on operational issues.
 
 
     scaStatusResponse:
       description: Body of the JSON response with SCA Status.
+      required: 
+        - scaStatus
```

Comment:
Relevant Change: scaStatus is now mandatory
Adoption approach: Follow the change

Reply:
already in there